src/EPR.Calculator.Frontend/Controllers/DashboardController.cs

```diff
@@ -27,35 +27,43 @@ public DashboardController(IConfiguration configuration, IHttpClientFactory clie
 
         public IActionResult Index()
         {
-            var dashboardcalculatorrunApi = this.configuration.GetSection("DashboardCalculatorRun").GetSection("DashboardCalculatorRunApi").Value;
-            var year = this.configuration.GetSection("DashboardCalculatorRun").GetSection("RunParameterYear").Value;
-            var client = this.clientFactory.CreateClient();
-            client.BaseAddress = new Uri(dashboardcalculatorrunApi);
-            var request = new HttpRequestMessage(HttpMethod.Post, new Uri(dashboardcalculatorrunApi));
-
-            var runParms = new CalculatorRunParamsDto
+            try
             {
-                FinancialYear = year,
-            };
-            var content = new StringContent(JsonConvert.SerializeObject(runParms), System.Text.Encoding.UTF8, "application/json");
-            request.Content = content;
-            var response = client.SendAsync(request);
-            response.Wait();
+                var dashboardcalculatorrunApi = this.configuration.GetSection("DashboardCalculatorRun").GetSection("DashboardCalculatorRunApi").Value;
```

Comment:
Again constants for DashboardCalculatorRun and DashboardCalculatorRunApi

Reply:
This is not possible as getting the below error:

_The expression being assigned to must be constant._

Reply:
No worries, Will see later